packages/profile-sync-controller/src/shared/encryption/encryption.ts

```diff
@@ -100,7 +100,7 @@ class EncryptorDecryptor {
         p: SCRYPT_p,
         dkLen: ALGORITHM_KEY_SIZE,
       },
-      randomBytes(SCRYPT_SALT_SIZE),
+      undefined,
```

Comment:
URGH what a painful bug.

I kind of want to capture caching tests on this - but we can do this in a later PR to keep you unblocked for extension hotfix.